decred/decred/dcr/vsp.py

```diff
@@ -1,22 +1,23 @@
 """
 Copyright (c) 2019, Brian Stafford
-Copyright (c) 2019, the Decred developers
+Copyright (c) 2019-2020, the Decred developers
 See LICENSE for details
 
 DcrdataClient.endpointList() for available endpoints.
 """
 
 import time
 
+from decred import DecredError
 from decred.crypto import crypto
 from decred.util import encode, tinyhttp
-from decred.util.encode import ByteArray
 
 from . import constants, nets, txscript
 
 
 # The duration purchase info is good for.
 PURCHASE_INFO_LIFE = constants.HOUR
+ByteArray = encode.ByteArray
```

Comment:
Why doing this instead of the import?

Reply:
Using several methods of the module encode. Is my language correct there? I think three others. Would it be better to impor them separately?
It usually turns out that ByteArray is the most used, so giving it a shorthand name. Not so very short, maybe BA would actually be better.
Is it confusing? No problem changing.

Reply:
`ByteArray` is short enough, no need to shorten it further. :slightly_smiling_face:

I actually prefer `encode.ByteArray`, definitely not saying to separately import other names from `encode`.

If you feel that `encode.ByteArray` is too long, I'd rather import just `ByteArray` separately:

```
from decred.util.encode import ByteArray
```

rather than adding the name afterwards:

```
ByteArray = encode.ByteArray
```